src/git.js

```diff
@@ -189,12 +190,11 @@ class Git {
 		}
 	}
 
-	async getBlobContent(objectSha) {
-		return await execCmd(
-			`git show ${ objectSha }`,
-			this.workingDir,
-			false // Do not trim the result
-		)
+	async getBlobBase64Content(file) {
+		const fileRelativePath = path.join(this.workingDir, file)
+		const fileContent = await fs.promises.readFile(fileRelativePath)
+
+		return fileContent.toString('base64')
```

Comment:
Instead of pushing the content of the file, this solution converts the file to base64. 

Also, I have replaced the way of getting the content of the file. Previously I used the `git show` command, but I found that this could be problematic because of the `exec` command buffer size, and also the encoding that this command output produces.